features/apps/src/main/feature/feature.xml

```diff
@@ -44,6 +44,7 @@
         <feature>ddf-branding</feature>
         <feature>platform-app</feature>
         <feature>admin-app</feature>
+		<feature>fabric8-karaf-checks</feature>
```

Comment:
❓ Formatting?

Reply:
Odd, that looked right in my text editor. Fixed, thanks Vina!